src/applications/vaos/covid-19-vaccine/redux/actions.js

```diff
@@ -109,7 +113,9 @@ export function getClinics({ facilityId, showModal = false }) {
       dispatch({ type: FORM_FETCH_CLINICS });
       clinics = await getAvailableHealthcareServices({
         facilityId,
-        typeOfCareId: TYPE_OF_CARE_ID,
+        typeOfCare: TYPES_OF_CARE.find(
```

Comment:
I changed this function to take an object, because we need to use different ids depending on the feature flag